install-open-shift.sh

```diff
@@ -30,25 +30,33 @@ if [[ "$VAULT_ADDR" == "" || "$CLUSTER_NAME" == "" || "$CLUSTER_REGION" == "" ||
   exit 1
 fi
 
+if [[ "$KUBECONFIG" == "" ]] ; then
```

Comment:
Hmm, does this satisfy the case where it uses the file in ~/.kube? Maybe just trial run kubectl version and see if it runs successfully? Would have to move the check for kubectl cmd up as well.

Reply:
Yes, you are right. The kubectl version is already executed when checking the prereqs.